tests/backend/specs/api/tidy.js

```diff
@@ -1,27 +1,16 @@
-const assert = require('assert');
-os = require('os'),
-fs = require('fs'),
-path = require('path'),
-TidyHtml = null,
-Settings = null;
-
-const npm = require('../../../../src/node_modules/npm/lib/npm.js');
+const assert = require('assert').strict;
+const os = require('os');
+const fs = require('fs');
+const path = require('path');
+const TidyHtml = require('../../../../src/node/utils/TidyHtml');
+const Settings = require('../../../../src/node/utils/Settings');
```

Comment:
Execution order might matter here: These might need to be `require()`d after npm is loaded. I'm not sure though.

Reply:
Afaik I tested it and it was working.  I seem to remember installing tidy and making sure I was getting output etc.

Reply:
That said, I didn't appear to fully lint the file so it's possible it was never properly fixed.

Reply:
It looks like the tidy tests are completely broken. I restored the behavior to what it was before.